docs/03-tutorials/00-serverless/svls-03-expose-function.md

```diff
@@ -168,6 +168,6 @@ Follows these steps:
 7. Check if you can access the Function by selecting the HTTPS link under the **Host** column for the newly created APIRule.
 
     >**CAUTION:** If you have a Minikube cluster, you must first add its IP address mapped to the hostname of the exposed Kubernetes Service to the `hosts` file on your machine.
-
+<!-- Minikube?? -->
```

Comment:
We should use k3d for Kyma 2.0 instead of Minikube.

Reply:
removed the section